tests/data/hostgaplugin/ext_conf.xml

```diff
@@ -1,94 +1,119 @@
-          <Uri>https://zrdfepirv2cdm03prdstr01a.blob.core.windows.net/7d89d439b79f4452950452399add2c90/Microsoft.OSTCLinuxAgent_Test_uscentraleuap_manifest.xml</Uri>
-          <Uri>https://ardfepirv2cdm03prdstr01a.blob.core.windows.net/7d89d439b79f4452950452399add2c90/Microsoft.OSTCLinuxAgent_Test_uscentraleuap_manifest.xml</Uri>
-        </Uris>
-        <Version>2.5.0.2</Version>
-      </GAFamily>
-    </GAFamilies>
-    <Location>CentralUSEUAP</Location>
-    <ServiceName>CRP</ServiceName>
-  </GuestAgentExtension>
-  <RequiredFeatures>
-    <RequiredFeature>
-      <Name>MultipleExtensionsPerHandler</Name>
-    </RequiredFeature>
-  </RequiredFeatures>
-  <StatusUploadBlob statusBlobType="BlockBlob">https://dcrcqabsr1.blob.core.windows.net/$system/edpxmal5j1.058b176d-445b-4e75-bd97-4911511b7d96.status?sv=2018-03-28&amp;sr=b&amp;sk=system-1&amp;sig=U4KaLxlyYfgQ%2fie8RCwgMBSXa3E4vlW0ozPYOEHikoc%3d&amp;se=9999-01-01T00%3a00%3a00Z&amp;sp=w</StatusUploadBlob>
-  <Plugins>
-    <Plugin name="Microsoft.Azure.Extensions.CustomScript" version="2.1.6" location="https://umsavwggj2v40kvqhc0w.blob.core.windows.net/5237dd14-0aad-f051-0fad-1e33e1b63091/5237dd14-0aad-f051-0fad-1e33e1b63091_manifest.xml" state="enabled" autoUpgrade="true" failoverlocation="https://umsanh4b5rfz0q0p4pwm.blob.core.windows.net/5237dd14-0aad-f051-0fad-1e33e1b63091/5237dd14-0aad-f051-0fad-1e33e1b63091_manifest.xml" runAsStartupTask="false" isJson="true" useExactVersion="true">
-      <additionalLocations>
-        <additionalLocation>https://umsafwzhkbm1rfrhl0ws.blob.core.windows.net/5237dd14-0aad-f051-0fad-1e33e1b63091/5237dd14-0aad-f051-0fad-1e33e1b63091_manifest.xml</additionalLocation>
-      </additionalLocations>
-    </Plugin>
-    <Plugin name="Microsoft.CPlat.Core.RunCommandHandlerLinux" version="1.2.0" location="https://umsavbvncrpzbnxmxzmr.blob.core.windows.net/f4086d41-69f9-3103-78e0-8a2c7e789d0f/f4086d41-69f9-3103-78e0-8a2c7e789d0f_manifest.xml" state="enabled" autoUpgrade="true" failoverlocation="https://umsawqtlsshtn5v2nfgh.blob.core.windows.net/f4086d41-69f9-3103-78e0-8a2c7e789d0f/f4086d41-69f9-3103-78e0-8a2c7e789d0f_manifest.xml" runAsStartupTask="false" isJson="true" useExactVersion="true">
-      <additionalLocations>
-        <additionalLocation>https://umsajbjtqrb3zqjvgb2z.blob.core.windows.net/f4086d41-69f9-3103-78e0-8a2c7e789d0f/f4086d41-69f9-3103-78e0-8a2c7e789d0f_manifest.xml</additionalLocation>
-      </additionalLocations>
-    </Plugin>
-  </Plugins>
-  <PluginSettings>
-    <Plugin name="Microsoft.Azure.Extensions.CustomScript" version="2.1.6">
-      <RuntimeSettings seqNo="0">
-        {
-          "runtimeSettings": [
-            {
-              "handlerSettings": {
-                "publicSettings": {"commandToExecute":"echo '09cd27e9-fbd6-48ad-be86-55f3783e0a23'"}
-              }
-            }
-          ]
+<Extensions version="1.0.0.0" goalStateIncarnation="10"><GuestAgentExtension xmlns:i="http://www.w3.org/2001/XMLSchema-instance">
```

Comment:
I updated the test data from an actual DCR run to get the latest changes in HostGAPlugin